### DISCUSSION TITLE
Getting some statistics like the number of blog posts, tags, ... in my Docusaurus v3 blog

### DISCUSSION THREAD

**Discussion by cavo789** [facebook/docusaurus#9712] [Q&A] [answered]
Hi all 

Did you know how I can (without counting manually one by one ;)) obtain some statistics about my docusaurus blog. For instance how many articles I've written, perhaps by month, how many tags have been created/referenced, which tag is the most referenced one, ...

Thanks a lot!

**Comment by slorber** [comment]
> For instance how many articles I've written

This page shows all articles: https://docusaurus.io/blog/archive

You can probably swizzle and render a `blogPosts.length`, and the date is included so you can also display posts by months if you prefer to.

> how many tags have been created/referenced, which tag is the most referenced one

See this page: https://docusaurus.io/blog/tags

**Reply by cavo789** [reply]
Thanks Sébastien.

I was not aware of the `/archive` page; nice one.

But note, I was more thinking about statistics page; something I can copy/paste in an spreadsheet f.i. 
More oriented figures than display.

* Number of blog post: 999
* Number of tags : 999
    * docusaurus: 2 articles
    * docker: 15 articles

Was just for my own curiosity and *end of year status* ("I've written xxx articles and the top three tags were xxx, yyy and zzz").

Nothing important ;)

**Reply by slorber** [reply, marked as answer]
If you only need this info once, you can easily count things manually or run a console dev script.

```js
document.querySelectorAll("#__docusaurus_skipToContent_fallback > main > section > div > div > div > ul > li")
```

> NodeList(20)

If you need it to stay visible year after year, you can swizzle and implement your own stats block on the archive page


---

What prevents you from implementing this yourself?

> I was more thinking about statistics page

So this is a feature request, but only you asked for this so far and you only plan to use it to fill a spreadsheet once. 

That does not seem worth it to build it, and you should probably just create a node script to collect the stats you need. 

But if you really care you can design such a page in terms of content and layout, and explain why we should build this, feel free to open a real feature request.

**Reply by cavo789** [reply]
(I was not asking you to create the feature; just asking if the feature exists. sorry if it was not clear enough)

Nice tip for the query selector; I would never have thought of it.
